angr/analyses/cfg/cfg_fast.py

```diff
@@ -1113,13 +665,33 @@ def _calc_entropy(data, size=None):
     #
 
     @property
-    def memory_data(self):
-        return self._memory_data
+    def graph(self):
+        return self._model.graph
 
     @property
     def _insn_addr_to_memory_data(self):
         l.warning('_insn_addr_to_memory_data has been made public and is deprecated. Please fix your code accordingly.')
-        return self.insn_addr_to_memory_data
+        return self._model.insn_addr_to_memory_data
+
+    @property
+    def _memory_data(self):
+        return self._model.memory_data
+
```

Comment:
just keep only one?

Reply:
I'm keeping both `_memory_data` and `memory_data` for compatibility concerns.